templates/pricing/devices.html

```diff
@@ -243,22 +353,31 @@ <h3 class="p-heading--4">Are your devices running Ubuntu Desktop or Server?</h3>
             <a href="/security/livepatch">Kernel Livepatch</a> service to avoid reboots
           </li>
           <li class="p-list__item is-ticked">
-            <a href="/security/compliance-automation">Security certification</a>
+            <a href="/security/certifications">Security certification</a>
           </li>
           <li class="p-list__item is-ticked">
-            24/7, open source <a href="/support">software support</a> for the full stack.
+            24/7, open source <a href="/support">software support</a> for the full stack
           </li>
-          <li class="p-list__item is-ticked">Trademark use</li>
+          <li class="p-list__item is-ticked"><a href="/legal/trademarks">Trademark use</a></li>
         </ul>
-
-        <a href="/embedding" class="p-button--positive">Learn more about the Embedding programme</a>
+        <div class="p-cta-block">
+          <a href="/embedding">Learn more about the Embedding programme</a>
+        </div>
       </div>
     </div>
   </section>
 
-  {% with colour="light", bordered="false" %}
-    {% include "shared/_call-sales.html" %}
-  {% endwith %}
+  <section class="p-section--deep">
+    <div class="u-fixed-width">
+      <hr class="p-rule" />
+      <div class="p-section">
+        <h2 class="p-text--small-caps">Any questions?</h2>
+      </div>
+      <p class="p-heading--2">
+        Call us +1 737 204 0291 (Americas), +44 203 656 5291 (RoW) or <a href="https://ubuntu.com/pricing/devices#get-in-touch">Contact us online</a>
```

Comment:
Something for @Sophie-32 probably, but I don't think these phone numbers are used anymore. I remember we had to scrub mentions of them on u and c.com because according to marketing no one was picking up the phone 😅

Reply:
Oh yeah! I remember that. Side note, do you also remember the number that was just some guys personal number (he at least worked for Canonical). I will wait for @Sophie-32 to respond on this one before merging, as this also appears on another couple pages.

Reply:
I don't remember that, but I wish I did 😆